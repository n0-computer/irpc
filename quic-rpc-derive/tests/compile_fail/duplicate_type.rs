use quic_rpc_derive::rpc_requests;

#[rpc_requests(Service, Msg)]
enum Enum {
    A(u8),
    B(u8),
}

fn main() {}