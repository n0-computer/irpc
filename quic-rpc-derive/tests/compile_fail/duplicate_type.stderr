error: Each variant must have a unique request type
 --> tests/compile_fail/duplicate_type.rs:4:1
  |
4 | / enum Enum {
5 | |     A(u8),
6 | |     B(u8),
7 | | }
  | |_^
